packages/web/docs/src/content/gateway/logging-and-error-handling.mdx

```diff
@@ -0,0 +1,305 @@
+---
+title: Logging & Error Handling
```

Comment:
My general take is that logging and error-handling are a bit different, but i do understand why it makes sense when i read the content :) thanks @ardatan